src/wallet/crypter.cpp

```diff
@@ -205,9 +205,11 @@ bool CCryptoKeyStore::Unlock(const CKeyingMaterial& vMasterKeyIn, const bool& fF
         vMasterKey = vMasterKeyIn;
         fDecryptionThoroughlyChecked = true;
         if(!fFirstUnlock && zwalletMain){
-            uint160 hashSeedMaster = pwalletMain->GetHDChain().masterKeyID;
+            CHDChain hdChain = pwalletMain->GetHDChain();
```

Comment:
I believe this one is accidentally copying. It'll be better if we use `auto const &hdChain = ...` instead.

Reply:
Agree.

Reply:
Updated, please re-review